model-formulation/dataset_generator.py

```diff
@@ -0,0 +1,265 @@
+import pandas as pd
+import sqlite3
+import statistics
+
+
+DATA_STORE = '/home/ps35324/gpu-sharing-scheduler/src/data-store/scheduler.db'
```

Comment:
Please don't use hard coded paths. Can you please convert them to arguments with defaults and examples?